code/_onclick/item_attack.dm

```diff
@@ -67,10 +67,10 @@
 
 	if(force && M == user && user.client)
 		user.client.give_award(/datum/award/achievement/misc/selfouch, user)
-
+	var/datum/autopsy_record/AR = new /datum/autopsy_record(src,"weapon",damtype,force,name,sharpness,autopsy_description,FALSE)

```

Comment:
does this really need to be a datum?

I'm worried about memory usage if we're adding datums to lists on every single click on a mob